kubeflow/example/readme.md

```diff
@@ -7,7 +7,17 @@ The main objective of this pipeline is to perform the model training for a drivi
 
 The pipeline is composed of the following components:
 
-### Downloading data
+###  Interacting with MinIO
+
+#### Connecting with MinIO
+First of all, it is necessary to connect with the [MinIO](https://minio.platform.flexigrobots-h2020.eu/) server. For this reason, the user has to introduce the identification credentials. For security reasons, these values are read from environment variables. Accordingly, an .env file has to be created inside the [example](kubeflow\example\.env) folder. The credentials values in the .env are: 
```

Comment:
- Suggestion: "connect with" -> "connect to"
- I think "[example](kubeflow\example\.env)" should be "[example](kubeflow\example)" since ".env" doesn't exist in the repo

Reply:
Done! LGFM